pdns/minicurl.hh

```diff
@@ -28,17 +28,22 @@
 #include "iputils.hh"
 // turns out 'CURL' is currently typedef for void which means we can't easily forward declare it
 
+using MiniCurlHeaders = std::map<std::string, std::string>;
```

Comment:
Perhaps that could be moved inside the `MiniCurl` class to avoid cluttering the global namespace?

Reply:
pushed